### ISSUE TITLE
Any future updates to this repo - like updating to Node 10.x

### ISSUE THREAD

**Issue by raparlasiva** [AnomalyInnovations/serverless-nodejs-starter#43]
Is this repo maintained for future updates like updating to use Node 10.x

**Comment by jayair** [comment]
It will be yes. The 10.x Lambda update isn't particularly well put together. We are waiting till it is more stable and performant.

**Comment by trivikr** [comment]
Any update on this request?
The support for Node.js v10 was added in [May 2019](https://aws.amazon.com/about-aws/whats-new/2019/05/aws_lambda_adds_support_for_node_js_v10/).

**Comment by jayair** [comment]
Yup we'll be moving it. There isn't a whole lot to change btw. The serverless-bundle plugin already works with it. We just need to test things a bit more before updating the starter.
